### PR TITLE
Add Info about Plausible

### PR DESCRIPTION
**Pull request by maltejur**
**What change does this pull request introduce?**
Add the new page `/plausible`, which explains how analytics are used on the site.

**Screenshots**\
i9oudasbnogbasopg

**Checklist**

- [x] I worked on a branch other than `main`.
- [x] My branch is up-to-date with the Upstream `main` branch.
- [x] I have fixed potential errors using `yarn lint`.
- [x] I ran `yarn build` to check everything still builds successfully.

### CODE REVIEW

**Comment by maltejur** [comment]
@Panquesito7 is the text ok?

**Comment by maltejur** [comment]
#### 📝Changed routes:


| [**/plausible**](https://website-8y2rcu4kn-thealgorithms.vercel.app/plausible) |
|:-:|
|<a href="https://website-8y2rcu4kn-thealgorithms.vercel.app/plausible">Screenshot of /plausible</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/LK3jp0CYI45zle6OvHrnt8WG5xFUXiXf7vvh3xaYiYaDdFvNhuxeQtf1MVuMXeproUg9TdwG-1mPPPTVjgEWTQ.png">(view full size)</a> |







Commit 0bd7379dd4e44ca0353b9d46838d1de7555237ed (https://website-8y2rcu4kn-thealgorithms.vercel.app).

File: public/locales/en/common.json [hunk 1]
```diff
@@ -61,5 +61,8 @@
   "algorithmMetaDescriptionExplained": "{algorithm} explained and implemented in {implementations}",
   "languageMetaDescription": "Algorithms implemented in {language}",
   "noResults": "No results for",
-  "emptySearch" : "Search for an algorithm"
+  "emptySearch": "Search for an algorithm",
+  "analyticsFooter": "Analytics",
+  "analyticsTitle": "Analytics on this Site",
+  "analyticsText": "We use [Plausible Analytics] on this site to collect anonymous usage data. We do that you see how many people are using our website and which are the most used features, the website does not in any way track individual visitors. All the data is in aggregate only. No personal data is collected and no cookies are used. You can view all the collected data here:"
```
Comment (Panquesito7) [inline public/locales/en/common.json]:
```suggestion
  "analyticsText": "We use [Plausible Analytics] on this site to collect anonymous usage data to see how many people are using our website, and which are the most used features. The website does not in any way track individual visitors. All the data is in aggregate only. No personal data is collected, and no cookies are used. You can view all the collected data here:"
```

**Review by Panquesito7 — APPROVED** [review]
Looks good. Great work, @maltejur. Thanks! 👍 🎉

**Pull request merged**